### PR TITLE
Mcafee Test Fail

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/18350

## Description
Try to take out of skipped

### CODE REVIEW

**Comment by orlichter1** [comment]
@ShahafBenYakir as I told you on slack. This seems to be a flaky integration :(

File: Tests/conf.json [hunk 1]
```diff
@@ -2111,6 +2109,8 @@
         "Tanium": "issue 15497",
         "Tenable.sc": "unstable instance",
         "Tenable.io": "Issue 16115",
+        "SlackV2": "Issue 19558",
```
Comment (ShahafBenYakir) [inline Tests/conf.json]: Is this part of mcaffe?

File: Tests/conf.json [hunk 2]
```diff
@@ -2111,6 +2109,8 @@
...
+        "McAfee Threat Intelligence Exchange": "Unstable instance.",
```
Comment (ShahafBenYakir) [inline Tests/conf.json]: Let’s replace the comment with the issue number (better tracking)

File: Tests/conf.json (hunk 1 above)
Comment (orlichter1) [reply Tests/conf.json]: no. I merged it from master and this was in there. It must have been removed in the process. I'll remove it.

**Comment by orlichter1** [comment]
@ShahafBenYakir Dont merge yet. 
 Gonna try one last thing. 
sorry

**Pull request merged**